packages/suite/src/views/settings/SettingsDevice/SettingsDevice.tsx

```diff
@@ -85,10 +87,8 @@ export const SettingsDevice = () => {
 
     const deviceModelInternal = device.features.internal_model;
 
-    const supportsDeviceAuthentication = [
-        DeviceModelInternal.T2B1,
-        DeviceModelInternal.T3T1,
```

Comment:
The `T3B1` is missing here (most likely by error). => Refactored to use the type-safe map so we know to add this param or new device next time.